core/server/worker/src/main/java/alluxio/worker/netty/AbstractReadHandler.java

```diff
@@ -250,8 +250,11 @@ protected boolean acceptMessage(Object object) {
    */
   private void incrementMetrics(long bytesRead) {
     Counter counter = mContext.getCounter();
-    Preconditions.checkState(counter != null);
+    Preconditions.checkState(counter != null, "counter");
+    Counter userCounter = mContext.getUserCounter();
+    Preconditions.checkState(userCounter != null, "userCounter");
     counter.inc(bytesRead);
```

Comment:
do we still need the original `counter` now that we have `userCounter`? Is `counter` always the sum of all `userCounter` values?